Instagram/ComposeViewController.m

```diff
@@ -0,0 +1,75 @@
+#import "ComposeViewController.h"
+#import "Post.h"
+
+@interface ComposeViewController ()
+
+@property (strong, nonatomic) IBOutlet UIImageView *photo;
+@property (strong, nonatomic) IBOutlet UITextView *caption;
+- (IBAction)didTapCancel:(UIBarButtonItem *)sender;
```

Comment:
Typically easier to read if properties, class methods, and instance methods are separated into groups by a newline.